src/Inciter/Transporter.C

```diff
@@ -329,7 +332,7 @@ Transporter::load( uint64_t nelem )
 }
 
 void
-Transporter::partition()
+Transporter::part()
```

Comment:
Based upon the comment description and code inside, something along the lines of initiatePartitioning()? Or even "reportMeshTiming()". Although the part_complete() at the bottom makes me confused as to whether this is actually the signal for the completion of partitioning? But the text suggests this was just mesh reading?